packages/core/src/components/ui/ProductShelf/ProductShelf.tsx

```diff
@@ -40,6 +44,7 @@ function ProductShelf({
   productCardConfiguration: { bordered, showDiscountBadge } = {},
   numberOfItems,
   itemsPerPage = 5,
+  taxesConfiguration = {},
```

Comment:
question: why was it necessary to initialize it empty here?

Reply:
It is just for secure, to avoid "access property of undefined"